concordium-sdk/src/main/java/com/concordium/sdk/Client.java

```diff
@@ -313,6 +315,25 @@ public Semver getVersion() {
         return new Semver(versionString);
     }
 
+    /**
+     * Get the information for the given smart contract instance in the given block.
+     * @param req {@link GetInstanceInfoRequest}
+     * @return {@link InstanceInfo}
+     * @throws JsonProcessingException When the returned JSON is not in the specified standard.
```

Comment:
I don't think we should start exposing these now i.e. `JsonProcessingException`. Instead simply throw an `IllegalArgumentException` as we do the other places.

Reply:
done